src/components/NavMobile/NavMobile.scss

```
@@ -0,0 +1,332 @@
+@import '../../styles//utilities/colors.scss';
+@import '../../styles/utilities/breakpoints.scss';
+@import '../../styles/utilities/fonts.scss';
+
+.nav-mobile {
+  -webkit-tap-highlight-color: transparent;
+  display: none;
+  overflow: hidden;
+}
```

You declare this in the next class declaration in the media query

```suggestion
.nav-mobile {
  -webkit-tap-highlight-color: transparent;
  display: none;
}
```